.changeset/long-dolls-shake.md

```diff
@@ -0,0 +1,8 @@
+---
+"@comet/admin-theme": patch
+---
+
+Override `padding-right` of `MuiTablePagination`
+
+This fixes the styling of the pagination select in `DataGrid`s.
+Previously, the text and icon stuck together.
```

Comment:
I think this would be a bit simpler and the details of the `padding` and exact component are not really relevant to users of comet. 

```suggestion
Fix spacing between page number and chevron icon in the pagination select of `DataGrid`
```